[export] suppress type error

suo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #119720

Differential Revision: [D53681243](https://our.internmc.facebook.com/intern/diff/D53681243/)

suo committed:
[export] suppress type error



Differential Revision: [D53681243](https://our.internmc.facebook.com/intern/diff/D53681243/)

[ghstack-poisoned]

kit1980 commented on torch/_export/serde/serialize.py:
```diff
@@ -252,7 +252,7 @@ def _reconstruct_fake_tensor(serialized_tensor_meta: bytes, is_parameter: bool)
-    fake_tensor = _CURRENT_DESERIALIZER.deserialize_tensor_meta(tensor_meta)
+    fake_tensor = _CURRENT_DESERIALIZER.deserialize_tensor_meta(tensor_meta)  # type: ignore[assignment]
     if is_parameter:
         fake_tensor = torch.nn.Parameter(fake_tensor)
```
The failure is in line 257, not 255.

suo committed:
Update on "[export] suppress type error"


Differential Revision: [D53681243](https://our.internmc.facebook.com/intern/diff/D53681243/)

[ghstack-poisoned]

suo commented:
@pytorchbot merge -f "fix lint"
